### ISSUE TITLE
importing failed

### ISSUE THREAD

**Issue by aminemosbah** [rdkit/rdkit-tutorials#9]
hi i get this when trying to run the tutorial, what to we need to install rdkit in google collab ? thinks
ImportError                               Traceback (most recent call last)
<ipython-input-3-bdf155927f74> in <module>()
      1 import pandas as pd
----> 2 from rdkit import Chem
      3 from rdkit.Chem import PandasTools
      4 from rdkit.Chem.Draw import IPythonConsole
      5 

ImportError: cannot import name 'Chem'

---------------------------------------------------------------------------
NOTE: If your import is failing due to a missing package, you can
manually install dependencies using either !pip or !apt.

To view examples of installing some common dependencies, click the
"Open Examples" button below.

**Comment by jw-feng** [comment]
Try installing RDKit with the following:

!wget -c
https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
!chmod +x Miniconda3-latest-Linux-x86_64.sh
!bash ./Miniconda3-latest-Linux-x86_64.sh -b -f -p /usr/local
!conda install -q -y -c conda-forge rdkit
# Then append rdkit path to current python system path.
import sys
sys.path.append('/usr/local/lib/python3.7/site-packages/')

**Comment by aminemosbah** [comment]
Are you sure this is the way to go with google collab ? I m trying to run rdkit on the cloud not on the local device

**Comment by jw-feng** [comment]
Yes, see this example that was posted on the Rdkit discussion forum. https://colab.research.google.com/drive/1nQPmdEbYQgVsFr7c44yRd3wpXPEsJar3

This is what worked for me. I tried installing RDKit with apt-get on colab but that resulted in an error trying to import Chem. 

JW

**Comment by aminemosbah** [comment]
Thanks that’s fine

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
